source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -175,6 +175,72 @@ class NiDAQmxDriverApiTests : public Test {
     return create_ai_voltage_chan(request, response);
   }
 
+  CreateAIBridgeChanRequest create_ai_bridge_request(double min_val, double max_val, const std::string& custom_scale_name = "")
+  {
+    CreateAIBridgeChanRequest request;
+    set_request_session_name(request);
+    request.set_physical_channel("bridgeTester/ai0");
+    request.set_name_to_assign_to_channel("ai0");
+    request.set_min_val(min_val);
+    request.set_max_val(max_val);
+    if (custom_scale_name.empty()) {
+      request.set_units(BridgeUnits::BRIDGE_UNITS_VOLTS_PER_VOLT);
+    }
+    else {
+      request.set_custom_scale_name(custom_scale_name);
+      request.set_units(BridgeUnits::BRIDGE_UNITS_FROM_CUSTOM_SCALE);
+    }
+    request.set_bridge_config(BridgeConfiguration1::BRIDGE_CONFIGURATION1_FULL_BRIDGE);
+    request.set_voltage_excit_source(ExcitationSource::EXCITATION_SOURCE_INTERNAL);
+    request.set_voltage_excit_val(2.50);
+    request.set_nominal_bridge_resistance(350.00);
+    return request;
+  }
+
+  ::grpc::Status create_ai_bridge_chan(const CreateAIBridgeChanRequest& request, CreateAIBridgeChanResponse& response = ThrowawayResponse<CreateAIBridgeChanResponse>::response())
+  {
+    ::grpc::ClientContext context;
+    auto status = stub()->CreateAIBridgeChan(&context, request, &response);
+    client::raise_if_error(status, context);
+    return status;
+  }
+
+  ::grpc::Status create_ai_bridge_chan(double min_val, double max_val, CreateAIBridgeChanResponse& response = ThrowawayResponse<CreateAIBridgeChanResponse>::response())
+  {
+    auto request = create_ai_bridge_request(min_val, max_val);
+    return create_ai_bridge_chan(request, response);
+  }
+
+  CreateAIThrmcplChanRequest create_ai_thrmcpl_request(double min_val, double max_val, const std::string& custom_scale_name = "")
+  {
+    CreateAIThrmcplChanRequest request;
+    set_request_session_name(request);
+    request.set_physical_channel("cDAQ1Mod2/ai0");
+    request.set_name_to_assign_to_channel("ai0");
+    request.set_min_val(min_val);
+    request.set_max_val(max_val);
+    request.set_units(TemperatureUnits::TEMPERATURE_UNITS_DEG_C);
+    request.set_thermocouple_type(ThermocoupleType1::THERMOCOUPLE_TYPE1_J_TYPE_TC);
+    request.set_cjc_val(25.0);
+    request.set_cjc_channel("");
+    request.set_cjc_source(CJCSource1::CJC_SOURCE1_CONST_VAL);
+    return request;
+  }
+
+  ::grpc::Status create_ai_thrmcpl_chan(const CreateAIThrmcplChanRequest& request, CreateAIThrmcplChanResponse& response = ThrowawayResponse<CreateAIThrmcplChanResponse>::response())
+  {
+    ::grpc::ClientContext context;
+    auto status = stub()->CreateAIThrmcplChan(&context, request, &response);
+    client::raise_if_error(status, context);
+    return status;
+  }
+
+  ::grpc::Status create_ai_thrmcpl_chan(double min_val, double max_val, CreateAIThrmcplChanResponse& response = ThrowawayResponse<CreateAIThrmcplChanResponse>::response())
```

Comment:
I'm not sure what's going on here.

```
D:\a\grpc-device\grpc-device\source\tests\system\nidaqmx_driver_api_tests.cpp(830,18): error C2535: 'grpc::Status ni::tests::system::`anonymous-namespace'::NiDAQmxDriverApiTests::create_ai_thrmcpl_chan(double,double,nidaqmx_grpc::CreateAIThrmcplChanResponse &)': member function already defined or declared [D:\a\grpc-device\grpc-device\build\SystemTestsRunner.vcxproj]
D:\a\grpc-device\grpc-device\source\tests\system\nidaqmx_driver_api_tests.cpp(238,18): message : see declaration of 'ni::tests::system::`anonymous-namespace'::NiDAQmxDriverApiTests::create_ai_thrmcpl_chan' [D:\a\grpc-device\grpc-device\build\SystemTestsRunner.vcxproj]
D:\a\grpc-device\grpc-device\source\tests\system\nidaqmx_driver_api_tests.cpp(962,43): error C2065: 'skip_unsupported_channels': undeclared identifier [D:\a\grpc-device\grpc-device\build\SystemTestsRunner.vcxproj]
D:\a\grpc-device\grpc-device\source\tests\system\nidaqmx_driver_api_tests.cpp(974,43): error C2065: 'skip_unsupported_channels': undeclared identifier [D:\a\grpc-device\grpc-device\build\SystemTestsRunner.vcxproj]
D:\a\grpc-device\grpc-device\source\tests\system\nidaqmx_driver_api_tests.cpp(2181,17): error C2660: 'ni::tests::system::`anonymous-namespace'::NiDAQmxDriverApiTests::perform_thrmcpl_lead_offset_nulling_cal': function does not take 1 arguments [D:\a\grpc-device\grpc-device\build\SystemTestsRunner.vcxproj]
D:\a\grpc-device\grpc-device\source\tests\system\nidaqmx_driver_api_tests.cpp(968,18): message : see declaration of 'ni::tests::system::`anonymous-namespace'::NiDAQmxDriverApiTests::perform_thrmcpl_lead_offset_nulling_cal' [D:\a\grpc-device\grpc-device\build\SystemTestsRunner.vcxproj]
D:\a\grpc-device\grpc-device\source\tests\system\nidaqmx_driver_api_tests.cpp(2181,17): message : while trying to match the argument list '(nidaqmx_grpc::PerformThrmcplLeadOffsetNullingCalResponse)' [D:\a\grpc-device\grpc-device\build\SystemTestsRunner.vcxproj]
```

Reply:
There was already a definition of this method down on line 830:
https://github.com/ni/grpc-device/blob/5c83a17e04c21745cf6f89039c8cce9a09370dbc/source/tests/system/nidaqmx_driver_api_tests.cpp#L830

At a glance they seem to be doing more or less the same thing so maybe we could just delete the method on line 830.

Reply:
Thanks for checking, I will remove the method on line 830.